### ISSUE TITLE
Can this be archived?

### ISSUE THREAD

**Issue by marcoscaceres** [WICG/encrypted-media#2]
There doesn't seem to have been much activity since we brought this to the WICG... can this be archived?

**Comment by mwatson2** [comment]
No, this is the main repo for the persistent-usage-record feature which forms part of the work of the new Media Working Group.

I interpret the lack of activity as being because the feature was basically fully developed during the EME v1 time period.

**Comment by marcoscaceres** [comment]
Ok, thanks fo confirming @mwatson2. Just wanted to check.

**Comment by marcoscaceres** [comment]
Hey, just wanted to check in again... the new media WG is up and running so we are wondering if now this is owned by them (and if we can archive this?) @mounirlamouri @mwatson2 what say you?

**Comment by mounirlamouri** [comment]
@mwatson2 had to merge this work to the W3C repo. If it has been done, I believe this repo can be deleted.

**Comment by mwatson2** [comment]
@gregwfreedman is working on that.

**Comment by marcoscaceres** [comment]
Any update here?

**Comment by joeyparrish** [comment]
This has been merged into the WD spec and can be archived.
